### ISSUE TITLE
Bug: must be an array or an object that implements Countable

### ISSUE THREAD

**Issue by orlov0562** [archetyped/simple-lightbox#646]
Hi there,

Faced with error:
```
must be an array or an object that implements Countable in /wp-content/plugins/simple-lightbox/includes/class.utilities.php on line 1411
```
looks like this error generated by the **count($_GET['page'])** in next line:
```
...
// Otherwise, Determine action based on plugin plugin admin page suffix
elseif ( isset($_GET['page']) && ($pos = strrpos($_GET['page'], '-')) && $pos !== false && ( $pos != count($_GET['page']) - 1 ) )
$action = trim(substr($_GET['page'], $pos + 1), '-_');
...
```
I think you should change **count** to **strlen** function

**Comment by HenkeSoldF** [comment]
I have the same problem .

**Comment by Beee4life** [comment]
Same here... but only on Nginx (afaik).

I have developed the site on Apache (where I don't see it) but on Nginx I do notice it. Not sure if the settings are 100% the same since someone just installed it for me a few minutes ago.

**Comment by Beee4life** [comment]
I found that the fix was easier than wrapping it in an is_admin().

On line 1411 change 
`$pos != count($_GET['page']) - 1`
to
`is_array( $_GET['page'] ) && $pos != count($_GET['page']) - 1`

**Comment by simonmiddleton** [comment]
Just to add the OP fix is correct for use on PHP 7.2, and is due to an incompatible change in that version. http://php.net/manual/en/migration72.incompatible.php

Patch attached if it helps.
[0001-Fix-PHP-7.2-incompatibility-in-Simple-Lightbox.patch.txt](https://github.com/archetyped/simple-lightbox/files/1916879/0001-Fix-PHP-7.2-incompatibility-in-Simple-Lightbox.patch.txt)
